packages/module/patternfly-docs/content/extensions/react-log-viewer/demos/ComplexToolbarLogViewer.jsx

```diff
@@ -181,10 +182,10 @@ const ComplexToolbarLogViewer = () => {
             isOpen={selectDataSourceOpen}
             placeholderText={selectDataSourcePlaceholder}
           >
-            {selectDataSourceMenu}
+            <SelectList>{selectDataSourceMenu}</SelectList>
```

Comment:
With the new implementation of Select a SelectList component has to be manually passed in, otherwise SelectOptions will render as `li` in the DOM without an `ul` parent.